### ISSUE TITLE
OS X 10.11: Can't move the window

### ISSUE THREAD

**Issue by lise-yy** [danielchatfield/trello-desktop#6]
Hi! Thank you for creating the desktop app for Trello. 

While using I found a bug. I can't move the app's window, [video](http://s.csssr.io/2016-07-07-1115-xlfocnts6d.mp4).

**Comment by janbrasna** [comment]
Yup, confirmed on latest 10.11.5/6 … Had to resize it to place the window elsewhere;)

**Comment by danielchatfield** [comment]
I'm also experiencing this but a little too busy to take a look right now.

**Comment by janbrasna** [comment]
This changed at some point earlier this year, refs: https://github.com/electron/electron/issues/5678

There's how Atom guys approached it: https://github.com/atom/atom/pull/10208

**Comment by rc1** [comment]
Seems like this is fixed in master. But the release download doesn't contain the fix.

**Comment by nicolasandrieux** [comment]
What is the plan to have it on the release?

**Comment by yurivecchi** [comment]
I can make a release, but I don't have a Mac to test and build

**Comment by diegotoral** [comment]
@yurivecchi I've built the Mac version. It seems pretty stable for release. I'm missing a change log file.

**Comment by danielchatfield** [comment]
@diegotoral I've made you a collaborator :)

**Comment by diegotoral** [comment]
Thanks @danielchatfield!
